draft-js-link-plugin/src/components/Link/index.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-param-reassign */
+import React, { PropTypes } from 'react';
+
+const propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node.isRequired,
+  entityData: PropTypes.object,
+  entityKey: PropTypes.string,
+  getEditorState: PropTypes.func
+};
+
+const Link = ({ children, entityData, entityKey, getEditorState, className }) => {
+  if (!entityData) {
+    if (!getEditorState) return <span />;
```

Comment:
Yep, that's true. There were some left over checks that I used in a similar component in order to work better with [draft-convert](https://github.com/HubSpot/draft-convert/). But they're actually not necessary here and only add noise. I've removed them now.